src/harness.js

```diff
@@ -205,37 +205,40 @@ class Harness {
   }
 
   /**
+   * @deprecated
```

Comment:
based on jsdoc you can actually write a comment along with the deprecation https://jsdoc.app/tags-deprecated.html

Can you indicate the version of deprecation and direct users to the replacement interface?